.travis.yml

```diff
@@ -51,15 +51,16 @@ install:
   - pip install dist/*.tar.gz
   - pip install h5py pillow
   - travis_wait pip install theano
-  - pip install nose hacking mock
+  - pip install pytest hacking mock
```

Comment:
Is it safe to remove nose dependency? I think existing tests still depend on it.

Reply:
OK, I added nose again.